test/fixtures/output/asyncapi/no-sr-key.golden

```diff
@@ -0,0 +1 @@
+Enter your Schema Registry API key: Error: EOF
```

Comment:
Don't we have a `--schema-registry-api-key` flag? I'd prefer that the user use that flag instead of typing it in.

Reply:
This is testing the scenario where the user has not passed any schema registry key. The scenario with the flag is being tested in other integration tests.

Reply:
Should the `--schema-registry-api-key` flag be required? I'd prefer if there was a single way to specify the API key.

Reply:
Alright, let's keep it for now, but I'll remove it in the next major release.